packages/calypso-build/bin/calypso-build.js

```diff
@@ -11,6 +11,8 @@
 const fs = require( 'fs' );
 const path = require( 'path' );
 
+process.env.BROWSERSLIST_ENV = process.env.BROWSERSLIST_ENV || 'wpcom';
```

Comment:
Interestingly this was missing here, but a similar logic is present in `transpile`.

If anybody was using `calypso-build` and not defining `BROWSERSLIST_ENV`, it would have used browserslist defaults. In our case, that default didn't match `fallback` nor `evergreen`.

Setting `BROWSERSLIST_ENV` here means that both `calypso-build` and `transpile` will use the same browserslist config.